src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -74,6 +74,7 @@ export default class DataTable extends Vue {
   cellWrap = Object.prototype.hasOwnProperty.call(this.$props.config, 'cellWrap')
     ? this.$props.config.cellWrap
     : defaultConfig.cellWrap;
+  showExpandAll = this.$props.config.showExpandAll || defaultConfig.showExpandAll;
```

Comment:
I see more of that kind of construction in this same file. It'd be a good idea for a future task to refactor some of these things :)